src/Cardano/Shell/Constants/Types.hs

```diff
@@ -226,21 +222,9 @@ data LastKnownBlockVersion = LastKnownBlockVersion
     -- ^ Last known block version alternative.
     } deriving (Eq, Show)
 
-data SSC = SSC
-    { sscMPCSendInterval               :: !Word
-      -- ^ Length of interval for sending MPC message
-    , sscMdNoCommitmentsEpochThreshold :: !Int
-      -- ^ Threshold of epochs for malicious activity detection
-    , sscNoReportNoSecretsForEpoch1    :: !Bool
-      -- ^ Don't print “SSC couldn't compute seed” for the first epoch.
-    } deriving (Eq, Show)
-
 data TXP = TXP
     { txpMemPoolLimitTx        :: !Int
       -- ^ Limit on the number of transactions that can be stored in the mem pool.
-    , txpAssetLockedSrcAddress :: ![Text]
-      -- ^ Set of source address which are asset-locked. Transactions which
-      -- use these addresses as transaction inputs will be silently dropped.
```

Comment:
Turns out this one is still needed for the Byron era. So this and all the other places above where this gets processed.